collect_app/src/androidTest/java/org/odk/collect/android/support/DisableDeviceAnimationsRule.kt

```diff
@@ -0,0 +1,29 @@
+package org.odk.collect.android.support
+
+import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
+import androidx.test.uiautomator.UiDevice
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class DisableDeviceAnimationsRule : TestRule {
```

Comment:
I'm **fairly** sure that animations are disabled on Firebase, but I saw some failures that were hard to explain otherwise. This also means we don't need to remember about switching animations off locally, so I think it's a net positive.